examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/lib/service_tier.py

```diff
@@ -122,6 +126,18 @@ def __init__(self, scope: Construct, stack_id: str, *, props: ServiceTierProps,
             version=props.deadline_version
         )
 
+        secrets_management_settings = None
+        if props.enable_secret_management:
+          if props.secret_management_secret_arn:
+            secrets_management_settings = {
+              "enabled": True,
+              "credentials": Secret.from_secret_arn(self, 'SMAdminUser', props.secret_management_secret_arn)
+            }
+        else:
+          secrets_management_settings = {
+            "enabled": False
+          }
```

Comment:
A few things to fix here:
1. The indentation looks like it's using 2 spaces instead of 4.
2. Testing the Optional `props.secret_management_secret_arn` should use `is not None`.
3. The two `if` statements could be combined with an `and` instead of nesting them.

It might be more readable to split it up this way:
```suggestion
        secrets_management_settings = {}
        secrets_management_settings["enabled"] = props.enable_secret_management
        if props.secret_management_secret_arn is not None:
            secrets_management_settings["credentials"] = Secret.from_secret_arn(self, 'SMAdminUser', props.secret_management_secret_arn)
```